### PR TITLE
[15.0][IMP] account_statement_import_online: Add debug wizard

### PR DESCRIPTION
**Pull request by pedrobaeza**
It's useful to debug RAW data received from your online sources. With this improvement, you can do it easily from the manual pull wizard. It's only available for users in debug mode (or with technical features).

@Tecnativa TT47238

### CODE REVIEW

File: account_statement_import_online/wizards/online_bank_statement_pull_wizard.xml [hunk 1]
```diff
@@ -28,6 +28,12 @@
                         default_focus="1"
                         class="btn-primary"
                     />
+                    <button
+                        name="action_debug"
+                        string="Debug"
```
Comment (StefanRijnhart) [inline account_statement_import_online/wizards/online_bank_statement_pull_wizard.xml]: I think the label on this one can be a little bit more descriptive (accountants also use debug mode to see more features, not just developers). Maybe `Show transaction data` plus a more explanatory help text on this one and then replace `Pull` on the other button by something like `Import transactions`?

**Review by StefanRijnhart — COMMENTED** [review]
Very useful! I'm porting this to 16.

**Review by carlosdauden — APPROVED** [review]

File: account_statement_import_online/wizards/online_bank_statement_pull_wizard.xml (hunk 1 above)
Comment (StefanRijnhart) [reply account_statement_import_online/wizards/online_bank_statement_pull_wizard.xml]: My suggestions here: https://github.com/stefanrijnhart/bank-statement-import/commit/6d104bd597bbde65a3542c81e365acb5b4b4e665

**Pull request merged**